app/src/main/java/hello.java

```diff
@@ -0,0 +1,5 @@
+public class hello {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Move this left curly brace to the beginning of next line of code. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://aa3286466891111e6982e120118ddea2-36576358.us-east-1.elb.amazonaws.com/sonar/coding_rules#rule_key=squid%3ALeftCurlyBraceStartLineCheck)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this empty class, write its code or make it an "interface". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://aa3286466891111e6982e120118ddea2-36576358.us-east-1.elb.amazonaws.com/sonar/coding_rules#rule_key=squid%3AS2094)
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this class name to match the regular expression '^[A-Z][a-zA-Z0-9]*$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://aa3286466891111e6982e120118ddea2-36576358.us-east-1.elb.amazonaws.com/sonar/coding_rules#rule_key=squid%3AS00101)